trigger.tex

```diff
@@ -58,10 +58,13 @@ \section{Actions}
     \Rmepc must contain the virtual address of the next instruction that must
     be executed to preserve the program flow. \\
 \hline
-1 & Enter Debug Mode. (Only legal when the trigger's \FcsrTdataOneDmode
-    is 1.)
+1 & Enter Debug Mode.
     \RcsrDpc must contain the virtual address of the next instruction that must
-    be executed to preserve the program flow. \\
+    be executed to preserve the program flow.
+
+    This action is only legal when the trigger's \FcsrTdataOneDmode is 1.
+    Hardware should ignore triggers which have this action configured without
+    having \FcsrTdataOneDmode set. \\
```

Comment:
Would it be better to say that hardware must not accept a write to mcontrol with dmode=0 and action=1.  rocket-chip forces action to 0 if dmode is being written with 0.